WcaOnRails/app/inputs/money_amount_input.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+class MoneyAmountInput < SimpleForm::Inputs::Base
+  def input(wrapper_options)
+    merged_input_options = merge_wrapper_options(input_html_options, wrapper_options)
+
+    actual_field = @builder.hidden_field(attribute_name)
+    input_id = attribute_name.to_s + "_input_field"
+    script = template.content_tag(:script, type: 'text/javascript') do
+      raw("wca.setupCurrencyMask('##{input_id}', '##{@builder.object_name}_#{attribute_name}', '#{currency}');")
```

Comment:
This works, but feels like it's happening in the wrong place. We create the hidden input here in Ruby, but we keep that hidden input up to date over in javascript land. I think it would be a nicer separation of concerns if we move the hidden input creation over to javascript.

I think we could achieve my desired separation of concerns by creating an input with some attributes here (say `<input type="currency" data-currency="#{currency}" ...`), and then there'd be some jquery that runs on page load that would look for `input[type="currency"]` and do all the magic of creating the hidden input with the appropriate `name` and `value` attributes.